src/blocktree.rs

```diff
@@ -123,20 +123,25 @@ pub trait LedgerColumnFamilyRaw {
 #[derive(Clone, Debug, Default, Deserialize, Serialize, Eq, PartialEq)]
 // The Meta column family
 pub struct SlotMeta {
+    // The number of slots above the root (the genesis block). The first
+    // slot has slot_height 0.
     pub slot_height: u64,
-    // The total number of consecutive blob starting from index 0
+    // The total number of consecutive blobs starting from index 0
     // we have received for this slot.
     pub consumed: u64,
-    // The entry height of the highest blob received for this slot.
+    // The index *plus one* of the highest blob received for this slot.
+    // Useful for checking if the slot has received any blobs yet.
```

Comment:
also useful for repair, scans for holes can limit themselves to received - consumed indexes...

Reply:
fixed